xlive/H2MOD/Utils/Utils.cpp

```diff
@@ -730,3 +730,91 @@ std::string ByteToHexStr(const uint8_t* buffer, size_t size) {
 	}
 	return str.str();
 }
+
+std::string base64_encode(const std::vector<unsigned char>& binaryData) {
```

Comment:
These functions are duplicates of the ones in Base64.cpp

Reply:
Functions are different, but I moved them into the same files and renamed them to base64_encode_binary and base64_decode_binary.